docs/conf.py

```diff
@@ -74,7 +74,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ['_build', '.*']
```

Comment:
This makes Sphinx ignore hidden files, which helps it work better in my workflow.